samples/Test.AppWriter/Test.AppWriter.csproj

```diff
@@ -30,9 +29,7 @@
 
   <ItemGroup>
     <!-- This reference will be replaced with Nuget package reference -->
-    <Reference Include="Microsoft.PowerPlatform.PowerApps.Persistence">
-      <HintPath>..\..\bin\Debug\Persistence\Microsoft.PowerPlatform.PowerApps.Persistence.dll</HintPath>
-    </Reference>
+  <ProjectReference Include="..\..\src\Persistence\Microsoft.PowerPlatform.PowerApps.Persistence.csproj" />
```

Comment:
Clarifying question: Will this change affect the build process for the sample apps? (i.e. will Persistence be rebuilt when building the Samples solution)

Reply:
it will be rebuilt if it has changes, msbuild will take care of it.

Reply:
there will be a slight general msbuild overhead, but no actual recompilation will happen.